### PR TITLE
Feature/fuse binary conv

### PR DESCRIPTION
**Pull request by zhen8838**
- [x] add merge pattern for add(conv(x),y)

---

Before:
截屏2023-01-13 13 37 44

---
After:
截屏2023-01-13 13 38 22

### CODE REVIEW

**Review by xhuohai — COMMENTED** [review]
LGTM

File: src/Nncase.Tests/Rewrite/Fusion/UnitTestFusionGroup.cs [hunk 1]
```diff
@@ -56,15 +65,18 @@ private void RunCore(IDataFlowFusionCase fusionCase)
 
         IRModule module = new(main);
         CompilerServices.InferenceType(main);
+        Dumpper.DumpDotIR(main, "pre");
```
Comment (sunnycase) [inline src/Nncase.Tests/Rewrite/Fusion/UnitTestFusionGroup.cs:68]: Disable dump in Release mode for speedup CI.

File: src/Nncase.Transform/Mutators/IFusionMergeRule.cs [hunk 2]
```diff
@@ -245,17 +233,31 @@ public virtual Pattern CreatePattern(string target_module_kind)
         }
 
         var callee_input = (Expr)result["callee_input"];
```
Comment (sunnycase) [inline src/Nncase.Transform/Mutators/IFusionMergeRule.cs:235]: Use camel casing.

File: src/Nncase.Tests/Rewrite/Fusion/UnitTestFusionGroup.cs (hunk 1 above)
Comment (zhen8838) [reply src/Nncase.Tests/Rewrite/Fusion/UnitTestFusionGroup.cs:69]: disabled

**Review by xhuohai — APPROVED** [review]

**Pull request merged**